packages/curriculum-compiler-json/compilers/exercise.js

```diff
@@ -0,0 +1,17 @@
+const { compileNodeToQuestionMarkdown } = require('./helpers');
+
+module.exports = function exercise(node) {
+  const rawText = compileNodeToQuestionMarkdown(node);
```

Comment:
This type of approach is an issue cause all other parser/compiler combos consider an exercise equivalent to an insight, not a question.

Reply:
`compileNodeToQuestionMarkdown` is specific to a question. There should be an insight equivalent version.

Reply:
Well, there's a difference between the Exercise file type and the Exercise section...
[Exercise files](https://github.com/enkidevs/curriculum/blob/master/javascript/core/functions-ii/js-practice-this.md) will have yaml metadata and a single Exercise section

Reply:
oooooooh, got it